docs/source/guides/understanding_types_and_tags.ipynb

```diff
@@ -383,13 +375,13 @@
    "source": [
     "### Index and Time Index Tags\n",
     "\n",
-    "When creating a DataTable, you have the option to specify which column represents the index and which column represents the time index. If these columns are specified, semantic tags of `index` and `time_index` are applied to the specified columns. Behind the scenes, Woodwork is performing additional validation checks on the columns to make sure they are appropriate. For example, index columns must be unique, and time index columns must contain datetime values or values that can be converted to datetimes.\n",
+    "When initializing Woodwork, you have the option to specify which column represents the index and which column represents the time index. If these columns are specified, semantic tags of `index` and `time_index` are applied to the specified columns. Behind the scenes, Woodwork is performing additional validation checks on the columns to make sure they are appropriate. For example, index columns must be unique, and time index columns must contain datetime values or values that can be converted to datetimes.\n",
```

Comment:
> time index columns must contain datetime values or values that can be converted to datetimes

We allow for numeric time indices as well as converting numeric columns to datetime values, so maybe this would be more correct if it was 

> time index columns must contain datetime or numeric values